apps/documentation/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "application",
+  "sourceRoot": "apps/documentation/src",
+  "tags": [],
+  "targets": {
+    "build": {
```

Comment:
I think it would make more sense to also call this `generate-docs`, so that it can have a declared dependency on the other `generate-docs` of the libs.

Because right now there is a hidden dependency, that you need to run `generate-docs` of all libs before running `build` or `serve` here.